File: src/main/java/io/github/benjohnde/ankeader/MinimalisticUI.java
package io.github.benjohnde.ankeader;

import javafx.application.Application;
import javafx.application.Platform;
import javafx.stage.FileChooser;
import javafx.stage.Stage;

import java.io.File;

public class MinimalisticUI extends Application {

    @Override
    public void start(Stage primaryStage) throws Exception {
        FileChooser fileChooser = new FileChooser();
        fileChooser.setTitle("Open Resource File");
        File retValue = fileChooser.showOpenDialog(primaryStage);
        Platform.exit();
    }
}

File: src/main/java/io/github/benjohnde/ankeader/MinimalisticUI.kt
package io.github.benjohnde.ankeader

import javafx.application.Application
import javafx.application.Platform
import javafx.stage.FileChooser
import javafx.stage.Stage

class MinimalisticUI : Application() {

    @Throws(Exception::class)
    override fun start(primaryStage: Stage) {
        val fileChooser = FileChooser()
        fileChooser.title = "Open Resource File"
        val retValue = fileChooser.showOpenDialog(primaryStage)
        println(retValue)
        Platform.exit()
    }
}
